build/configs/esp32_DevKitC/kernel_tc/defconfig

```diff
@@ -0,0 +1,839 @@
+# CONFIG_EXAMPLES_EEPROM_TEST is not set
+# CONFIG_EXAMPLES_EVENTLOOP is not set
+# CONFIG_EXAMPLES_FOTA_SAMPLE is not set
+# CONFIG_FILESYSTEM_HELPER_ENABLE is not set
+CONFIG_EXAMPLES_HELLO=y
+# CONFIG_EXAMPLES_HELLOXX is not set
+# CONFIG_EXAMPLES_IOTBUS_TEST is not set
+# CONFIG_EXAMPLES_IOTJS_STARTUP is not set
+# CONFIG_EXAMPLES_KERNEL_SAMPLE is not set
+# CONFIG_EXAMPLES_LIBTUV is not set
+# CONFIG_EXAMPLES_NETTEST is not set
+# CONFIG_EXAMPLES_PROC_TEST is not set
+# CONFIG_EXAMPLES_SELECT_TEST is not set
+# CONFIG_EXAMPLES_SENSORBOARD is not set
+# CONFIG_EXAMPLES_SETJMP_TEST is not set
+# CONFIG_EXAMPLES_SMART is not set
+# CONFIG_EXAMPLES_SMART_TEST is not set
+# CONFIG_EXAMPLES_SPEECH_DETECTOR_TEST is not set
+# CONFIG_EXAMPLES_ST_THINGS is not set
+CONFIG_EXAMPLES_TESTCASE=y
+# CONFIG_DISABLE_MANUAL_TESTCASE is not set
+# CONFIG_EXAMPLES_TESTCASE_DRIVERS is not set
+# CONFIG_EXAMPLES_TESTCASE_FILESYSTEM is not set
+CONFIG_EXAMPLES_TESTCASE_KERNEL=y
+# CONFIG_DEBUG_TC_KN is not set
+# CONFIG_TC_KN_ALL is not set
+CONFIG_TC_KERNEL_CLOCK=y
+CONFIG_TC_KERNEL_ENVIRON=y
+CONFIG_TC_KERNEL_ERRNO=y
+# CONFIG_TC_KERNEL_GROUP is not set
+CONFIG_TC_KERNEL_LIBC_FIXEDMATH=y
+CONFIG_TC_KERNEL_LIBC_INTTYPES=y
+CONFIG_TC_KERNEL_LIBC_LIBGEN=y
+CONFIG_TC_KERNEL_LIBC_MATH=y
+CONFIG_TC_KERNEL_LIBC_MISC=y
+CONFIG_TC_KERNEL_LIBC_MQUEUE=y
+CONFIG_TC_KERNEL_LIBC_PTHREAD=y
+CONFIG_TC_KERNEL_LIBC_QUEUE=y
+CONFIG_TC_KERNEL_LIBC_SCHED=y
+CONFIG_TC_KERNEL_LIBC_SEMAPHORE=y
+CONFIG_TC_KERNEL_LIBC_SIGNAL=y
+CONFIG_TC_KERNEL_LIBC_STDIO=y
+CONFIG_TC_KERNEL_LIBC_STDLIB=y
+CONFIG_TC_KERNEL_LIBC_STRING=y
+CONFIG_TC_KERNEL_LIBC_TIMER=y
+CONFIG_TC_KERNEL_LIBC_UNISTD=y
+CONFIG_TC_KERNEL_LIBC_SYSLOG=y
+CONFIG_TC_KERNEL_MQUEUE=y
+CONFIG_TC_KERNEL_PTHREAD=y
+# CONFIG_TC_KERNEL_ROUNDROBIN is not set
+CONFIG_TC_KERNEL_SCHED=y
+CONFIG_TC_KERNEL_SEMAPHORE=y
+CONFIG_TC_KERNEL_SIGNAL=y
+CONFIG_TC_KERNEL_TASK=y
+# CONFIG_TC_KERNEL_TERMIOS is not set
+CONFIG_TC_KERNEL_TIMER=y
+CONFIG_TC_KERNEL_UMM_HEAP=y
+# CONFIG_TC_KERNEL_TASH_HEAPINFO is not set
+# CONFIG_TC_KERNEL_TASH_STACKMONITOR is not set
+CONFIG_TC_KERNEL_WORK_QUEUE=y
+# CONFIG_TC_KERNEL_MEMORY_SAFETY is not set
+CONFIG_ITC_KERNEL_ENVIRON=y
+CONFIG_ITC_ENV_LOOP_SIZE=100
+CONFIG_ITC_KERNEL_LIBC_PTHREAD=y
+CONFIG_ITC_LIBC_PTHREAD_LOOP_SIZE=100
+CONFIG_ITC_KERNEL_LIBC_SEMAPHORE=y
+CONFIG_ITC_KERNEL_SEMAPHORE=y
+CONFIG_ITC_KERN_SEM_LOOP_SIZE=100
+CONFIG_ITC_KERNEL_SCHED=y
+CONFIG_ITC_KERNEL_TIMER=y
+CONFIG_ITC_KERNEL_PTHREAD=y
+CONFIG_ITC_KERN_PTHREAD_LOOP_SIZE=100
+# CONFIG_EXAMPLES_TESTCASE_NETWORK is not set
+# CONFIG_EXAMPLES_TESTCASE_TTRACE is not set
+# CONFIG_EXAMPLES_TESTCASE_ARASTORAGE_UTC is not set
+# CONFIG_EXAMPLES_TESTCASE_ARASTORAGE_ITC is not set
+# CONFIG_EXAMPLES_TESTCASE_AUDIO_UTC is not set
+# CONFIG_EXAMPLES_TESTCASE_AUDIO_ITC is not set
+# CONFIG_EXAMPLES_TESTCASE_SYSTEMIO_UTC is not set
+# CONFIG_EXAMPLES_TESTCASE_SYSTEMIO_ITC is not set
+# CONFIG_EXAMPLES_TESTCASE_WIFI_MANAGER_UTC is not set
```

Comment:
If it is only for kernel TCs,  let's remove unrelated CONFIG.

Reply:
It's created by make menuconfig, so I think it's better to keep all the items.